compatibility/BUILD

```diff
@@ -79,12 +79,12 @@ config_setting(
     if versions.is_at_least(sdk_version, platform_version)
 ]
 
-last_pre_7600_trigger_version = "1.7.0-snapshot.20201006.5358.0.0c1cadcf"
```

Comment:
We should figure out a way to compile the DAML model conditionally. We aren’t actually breaking backwards compatibility here so medium-term we shouldn’t have to change anything here.

Reply:
Fine to change it in this PR.

Reply:
There's a bit of a tarpit here, because we're conflating source compatibility (which e.g. this PR breaks) and SDK/runner compatibility (which e.g. this PR does not) in how we set up the cross builds. _And then_ you consider the divergent 1.6.0 series versus the 1.7.0 series, which have different requirements for each of these categories. Certainly worth some wonky Python [in a separate PR 🙂 ]